nuxeo-runtime/nuxeo-runtime-test/src/main/java/org/nuxeo/runtime/test/NXRuntimeTestCase.java

```diff
@@ -21,6 +21,7 @@
 import static org.junit.Assert.assertEquals;
 import static org.junit.Assert.assertNotNull;
 import static org.junit.Assert.fail;
+import static org.nuxeo.runtime.osgi.OSGiRuntimeService.getComponentsList;
```

Comment:
Please don't `import static` this method, it's surprising for the reader (which will look for it locally). Just `import org.nuxeo.runtime.osgi.OSGiRuntimeService` and call it as `OSGiRuntimeService.getComponentsList(...)`